db/migrate/20201020145209_create_giraffes.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :giraffes do |t|
       t.string :name, null: false
       t.text :description, null: false
+      t.belongs_to :user, null: false 
```

Comment:
This ideally should have been done through a migration, like how you `add user to review`